catalog/core/catalog-core-solr/src/test/java/ddf/catalog/source/solr/DynamicSchemaResolverTest.java

```diff
@@ -274,40 +274,50 @@ private MetacardType deserializeMetacardType(byte[] serializedMetacardType) thro
   @Test
   public void getField() {
     assertThat(
-        new DynamicSchemaResolver().getField("unknown", AttributeFormat.GEOMETRY, false),
+        new DynamicSchemaResolver()
+            .getField("unknown", AttributeFormat.GEOMETRY, false, Collections.EMPTY_MAP),
```

Comment:
should there be tests for passing in a map that isn't empty?

Reply:
Yes, forgot to add that.  I'll update the test to cover that case.